Makefile

```diff
@@ -28,7 +28,7 @@ release: build build-init
 debug: main-debug init-debug
 
 .PHONY: build
-build: linux-plugins
```

Comment:
build should still build linux-plugins?

Reply:
Build dockerfile/image does not depend on the linux-plugins image. Actually only the release and main-release depends on the "linux plugins"

It was slowing down my fluent-bit build tests.